### DISCUSSION TITLE
Custom fields in `directus_fields` doesn't show up in App when editing/creating a field

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#16432] [Feature Requests]
Directus isn't showing custom fields added to `directus_fields` like `directus_collections` does at the end of the collection page.

I was expecting see them in `Field` section when editing/creating a field. I'm going to use these custom fields as an additional field metadata for a code-generation extension (like adding a toggle `codegen_skip` to skip a field from code generation).

### To Reproduce

- Add a custom field to `directus_fields`
- Open or create a field in any collection
  - Custom field never shows up

**Comment by azrikahar** [comment]
> I was expecting see them in `Field` section when editing/creating a field. 

If this is where the expected new custom field(s) should appear:

![](https://user-images.githubusercontent.com/42867097/201566209-09702580-6d25-4786-9200-7af3ee0ac4eb.png)

Then I believe custom fields was never surfaced over here, so this will be an additional enhancement. One slightly tricky part might be to dynamically sync the custom fields to the field detail store. Relevant file for the intended tab where the custom field is expected to appear in App: https://github.com/directus/directus/blob/main/app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-field.vue

In the meantime it should still work via the API for a field's `meta.<custom_field>`:

![](https://user-images.githubusercontent.com/42867097/201566418-8f5c9367-01df-482a-8534-33e69c68b6f9.png)

**Reply by WoLfulus** [reply]
I landed once again in this issue in fields. Now I was expecting to be able to set a JSON Schema for my fields that are of type JSON and use that to once again, code-gen my client libraries. Would be nice to have this once (even if it's in a "custom" tab). 

Since we're able to create custom fields in system collections, it would be nice to have a place to set those values on all the collections. For example I also tried to add a custom field in "revisions" expecting it to show up whenever I open a revision and nope, nothing. 😢 

In this case I was trying to add a "status" to a revision, so I can "approve" some revisions and run a workflow on top of it.

Collections, files and users on the other hand, works beautifully well since they show up in the bottom of the respective detail page.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
